Rakefile

```diff
@@ -36,3 +36,36 @@ if File.exist?(build_defs_file)
     end
   end
 end
+
+desc "verify that commit messages match CONTRIBUTING.md requirements"
+task(:commits) do
+  commits = ENV['TRAVIS_COMMIT_RANGE'] || 'master..HEAD'
```

Comment:
I was worried this was going to end up mangled with the amount of merging we do between branches.